hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanCreateProjectTask.groovy

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.linkedin.gradle.azkaban;
+
+import org.apache.http.client.utils.URIBuilder;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.TaskAction;
+
+import org.json.JSONObject;
+
+class AzkabanCreateProjectTask extends DefaultTask {
+
+  AzkabanProject azkProject;
+
+  /**
+   * The Gradle task action for creating project in Azkaban.
+   */
+  @TaskAction
+  void create() {
+    if (azkProject.azkabanUrl == null) {
+      throw new GradleException("Please set azkaban.url in the .azkabanPlugin.json file in your project directory.");
+    }
+    createAzkabanProject(AzkabanHelper.readSession());
+  }
+
+  /**
+   * Creates a project in Azkaban from project name in .azkabanPlugin.json file.
+   *
+   * @param sessionId The Azkaban session id. If this is null, an attempt will be made to login to Azkaban.
+   */
+  void createAzkabanProject(String sessionId) {
+
+    sessionId = AzkabanHelper.resumeOrGetSession(sessionId, azkProject);
+
+    try {
+      //Fetch flows of the project
+
+      logger.lifecycle("Enter Project description:");
+      String description = System.console().readLine(" Input>");
+      while (description.isEmpty()) {
+        logger.lifecycle("Enter a non-empty Project description");
+        description = System.console().readLine(" Input>");
+      }
+
+      URI createURI = new URIBuilder(azkProject.azkabanUrl)
+          .setPath("/manager")
+          .setParameter("session.id", sessionId)
+          .setParameter("action", "create")
+          .setParameter("name", azkProject.azkabanProjName)
+          .setParameter("description", description.trim().toString())
+          .build();
+
+      String response = AzkabanHelper.responseFromPOST(createURI);
+
+      if (response.toLowerCase().contains("error")) {
```

Comment:
Similar to above comment, can we have an AzkabanResponses enum or class for responses such as error etc?